controllers/auth.go

```diff
@@ -276,7 +258,7 @@ func (c *ApiController) Login() {
 				//	object.LinkMemberAccount(userId, "avatar", avatar)
 				//}
 
-				resp = c.HandleLoggedIn(application, user, &form)
+				c.HandleLoggedIn(application, user, &form)
```

Comment:
You didn't save the result, it will be overwritten by: `c.Data["json"] = Response{Status: "null"}`